>From 9d244c3e4ebdd5e68ff75eee5b8bc4d654dc8277 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 11 Dec 2011 13:48:51 +0200
Subject: [PATCH] apt-file.1.sgml: Order items alphabetically
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 apt-file.1.sgml |  220 +++++++++++++++++++++++++++++-------------------------
 1 files changed, 118 insertions(+), 102 deletions(-)

diff --git a/apt-file.1.sgml b/apt-file.1.sgml
index bda42c4..31e35b2 100644
--- a/apt-file.1.sgml
+++ b/apt-file.1.sgml
@@ -52,56 +52,50 @@
     <para><command>apt-file</command> is a command line tool for
       searching files in packages for the APT package management system.</para>
     <para>Some actions are required to run the search:</para>
-
     <variablelist>
+
       <varlistentry>
-	<term>update</term>
+	<term>find</term>
 	<listitem>
 	  <para>
-	    Resynchronize the package contents from their sources. The
-	    lists of the contents of packages are fetched from the location(s)
-	    specified in
-	    <filename>/etc/apt/sources.list</filename>. This command
-	    attempts to fetch the
-	    <filename>Contents-&lt;ARCH&gt;.gz</filename> files from
-	    remote sources. For downloading these uses either the curl or
-	    wget commands as specified in <filename>apt-file.conf</filename>.
+	    Alias for <option>search</option>.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
-	<term>search</term>
+	<term>list</term>
 	<listitem>
 	  <para>
-	    Search in which package a file is included. A list of all
-	    packages containing the <option>pattern</option> pattern
-	    is returned.
+	    List the contents of a package matching the
+	    <option>pattern</option> pattern. This action is very
+	    close to the <command>dpkg -L</command> command except the
+	    package does not need to be installed or fetched.
 	  </para>
-	  <para>
-	    apt-file will only search for filenames, not directory names.
-	    This is due to the format of the Contents files it searches.
 	</listitem>
       </varlistentry>
 
       <varlistentry>
-	<term>find</term>
+	<term>purge</term>
 	<listitem>
 	  <para>
-	    Alias for <option>search</option>.
+	    remove all <filename>Contents-*</filename>
+	    files from the cache directory.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
-	<term>list</term>
+	<term>search</term>
 	<listitem>
 	  <para>
-	    List the contents of a package matching the
-	    <option>pattern</option> pattern. This action is very
-	    close to the <command>dpkg -L</command> command except the
-	    package does not need to be installed or fetched.
+	    Search in which package a file is included. A list of all
+	    packages containing the <option>pattern</option> pattern
+	    is returned.
 	  </para>
+	  <para>
+	    apt-file will only search for filenames, not directory names.
+	    This is due to the format of the Contents files it searches.
 	</listitem>
       </varlistentry>
 
@@ -115,49 +109,63 @@
       </varlistentry>
 
       <varlistentry>
-	<term>purge</term>
+	<term>update</term>
 	<listitem>
 	  <para>
-	    remove all <filename>Contents-*</filename>
-	    files from the cache directory.
+	    Resynchronize the package contents from their sources. The
+	    lists of the contents of packages are fetched from the location(s)
+	    specified in
+	    <filename>/etc/apt/sources.list</filename>. This command
+	    attempts to fetch the
+	    <filename>Contents-&lt;ARCH&gt;.gz</filename> files from
+	    remote sources. For downloading these uses either the curl or
+	    wget commands as specified in <filename>apt-file.conf</filename>.
 	  </para>
 	</listitem>
       </varlistentry>
+
     </variablelist>
   </refsect1>
 
   <refsect1>
     <title>OPTIONS</title>
     <variablelist>
+
       <varlistentry>
 	<term>
-	  <option>--cache | -c</option> <literal>cache-directory</literal>
+	  <option>-a, --architecture</option>
+	  architecture
 	</term>
 	<listitem>
 	  <para>
-	    Sets the cache directory to <filename>cache-directory</filename>
-	    instead of its default. If executed as non-root user, the default
-	    is <filename>$HOME/.cache/apt-file</filename> with fall-back to
-	    <filename>/var/cache/apt/apt-file</filename>. The latter is also
-	    the default if apt-file is called as root.
+	  Sets architecture to <literal>architecture</literal>. This
+	  option is useful if you search a package for a different
+	  architecture from the one installed on your system.
+	  It determines how the <literal>$ARCH</literal> variable
+	  in sources.list is expanded (but it does not influence
+	  the search in any other way).
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--verbose | -v</option>
+	  <option>-c, --cache</option> <literal>cache-directory</literal>
 	</term>
 	<listitem>
 	  <para>
-	    Run <command>apt-file</command> in verbose mode.
+	    Sets the cache directory to <filename>cache-directory</filename>
+	    instead of its default. If executed as non-root user, the default
+	    is <filename>$HOME/.cache/apt-file</filename> with fall-back to
+	    <filename>/var/cache/apt/apt-file</filename>. The latter is also
+	    the default if apt-file is called as root.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--cdrom-mount | -d</option> <literal>cdrom-mount-point</literal>
+	  <option>-d, --cdrom-mount</option> <literal>cdrom-mount-point</literal>
 	</term>
 	<listitem>
 	  <para>
@@ -169,121 +177,118 @@ Use <literal>cdrom-mount-point</literal> instead of
 
       <varlistentry>
 	<term>
-	  <option>--non-interactive | -N</option>
+	  <option>-D, --from-deb</option>
 	</term>
 	<listitem>
 	  <para>
-	  Skip schemes that are listed in the <literal>interactive</literal> line in
-	  <filename>apt-file.conf</filename>.
-	  This is useful if you want to call 'apt-file update' in cron jobs and skip all
-	  schemes that may require user input. 
+	    Use contents of the given .deb archives(s) as patterns.
+	    Useful for searching for file conflicts with other packages.
+	    Implies <option>-F</option>.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--ignore-case | -i</option>
+	  <option>-f, --from-file</option>
 	</term>
 	<listitem>
-	  <para>Ignore case when searching for <literal>pattern</literal>.</para>
+	  <para>
+	    Read patterns from the given file(s), one per line.
+	    Use <option>-f -<option> for stdin.
+	    This is much faster than invoking apt-file many times.
+	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--regexp | -x</option>
+	  <option>-F, --fixed-string</option>
 	</term>
 	<listitem>
 	  <para>
-	    Treat <literal>pattern</literal> as a (perl) regular expression. See
-	    <command>perlreref</command>(1) for details. Without this option,
-	    <literal>pattern</literal> is treated as a literal string to search
-	    for.
+	    Do not expand search pattern with generic characters at
+	    pattern's start and end.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--architecture | -a</option>
-	  architecture
+	  <option>-i, --ignore-case</option>
 	</term>
 	<listitem>
-	  <para>
-	  Sets architecture to <literal>architecture</literal>. This
-	  option is useful if you search a package for a different
-	  architecture from the one installed on your system.
-	  It determines how the <literal>$ARCH</literal> variable
-	  in sources.list is expanded (but it does not influence
-	  the search in any other way).
-	  </para>
+	  <para>Ignore case when searching for <literal>pattern</literal>.</para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--sources-list</option> | <option>-s</option>
-	  sources.list
+	  <option>-l, --package-only</option>
 	</term>
 	<listitem>
 	  <para>
-	    Sets the <filename>sources.list</filename> file to a
-	    different value from its default
-	    <filename>/etc/apt/sources.list</filename>.
+	    Only display package name; do not display file names.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--package-only | -l</option>
+	  <option>-N, --non-interactive</option>
 	</term>
 	<listitem>
 	  <para>
-	    Only display package name; do not display file names.
+	  Skip schemes that are listed in the <literal>interactive</literal> line in
+	  <filename>apt-file.conf</filename>.
+	  This is useful if you want to call 'apt-file update' in cron jobs and skip all
+	  schemes that may require user input.
 	  </para>
 	</listitem>
       </varlistentry>
 
       <varlistentry>
 	<term>
-	  <option>--from-file | -f</option>
+	  <option>-s, --sources-list</option>
+	  sources.list
 	</term>
 	<listitem>
 	  <para>
-	    Read patterns from the given file(s), one per line.
-	    Use <option>-f -<option> for stdin.
-	    This is much faster than invoking apt-file many times.
+	    Sets the <filename>sources.list</filename> file to a
+	    different value from its default
+	    <filename>/etc/apt/sources.list</filename>.
 	  </para>
 	</listitem>
       </varlistentry>
+
       <varlistentry>
 	<term>
-	  <option>--from-deb | -D</option>
+	  <option>-v, --verbose</option>
 	</term>
 	<listitem>
 	  <para>
-	    Use contents of the given .deb archives(s) as patterns.
-	    Useful for searching for file conflicts with other packages.
-	    Implies <option>-F</option>.
+	    Run <command>apt-file</command> in verbose mode.
 	  </para>
 	</listitem>
       </varlistentry>
+
       <varlistentry>
 	<term>
-	  <option>--fixed-string | -F</option>
+	  <option>-x, --regexp</option>
 	</term>
 	<listitem>
 	  <para>
-	    Do not expand search pattern with generic characters at
-	    pattern's start and end.
+	    Treat <literal>pattern</literal> as a (perl) regular expression. See
+	    <command>perlreref</command>(1) for details. Without this option,
+	    <literal>pattern</literal> is treated as a literal string to search
+	    for.
 	  </para>
 	</listitem>
       </varlistentry>
+
       <varlistentry>
 	<term>
-	  <option>--dummy | -y</option>
+	  <option>-y, --dummy</option>
 	</term>
 	<listitem>
 	  <para>
@@ -291,16 +296,19 @@ Use <literal>cdrom-mount-point</literal> instead of
 	  </para>
 	</listitem>
       </varlistentry>
+
       <varlistentry>
 	<term>
-	  <option>--help | -h</option>
+	  <option>-h, --help</option>
 	</term>
 	<listitem>
 	  <para>Display a short help screen.</para>
 	</listitem>
       </varlistentry>
+
     </variablelist>
   </refsect1>
+
   <refsect1>
     <title>CONFIGURATION FILE</title>
     <para>
@@ -312,6 +320,7 @@ Use <literal>cdrom-mount-point</literal> instead of
 	expansion</literal> section.
     </para>
     <variablelist>
+
       <varlistentry>
 	<term>destination</term>
 	<listitem>
@@ -320,6 +329,7 @@ Use <literal>cdrom-mount-point</literal> instead of
 	  </para>
 	</listitem>
       </varlistentry>
+
       <varlistentry>
 	<term>http | ftp | ssh | rsh | file | cdrom</term>
 	<listitem>
@@ -328,9 +338,11 @@ Use <literal>cdrom-mount-point</literal> instead of
 	  </para>
 	</listitem>
       </varlistentry>
+
     </variablelist>
     <refsect2>
       <title>String expansion</title>
+
       <para>
 	A <filename>sources.list</filename> entry is defined as:
 	<screen>
@@ -341,6 +353,7 @@ Use <literal>cdrom-mount-point</literal> instead of
 	  proto:/[/][user[:password]@]host[:port][/path]
 	</screen>
       </para>
+
       <variablelist>
 	<varlistentry>
 	  <term>&lt;host&gt;</term>
@@ -416,8 +429,31 @@ Use <literal>cdrom-mount-point</literal> instead of
   </refsect1>
 
   <refsect1>
-    <title>FILES</title>
+    <title>BUGS</title>
+    <para>The cdrom backend has not been tested.</para>
+    <para>
+      Non-release lines in sources.list are not handled by apt-file.
+    </para>
+    <para>
+     There is only one Contents file per distribution that contains all
+     components (i.e. main, contrib, and non-free). Threrefore, apt-file will
+     display search results from all components, even if not all components
+     are included in the sources.list file.
+    </para>
+    <para>
+     When a new line has been added to the sources.list and apt-file update has
+     not been run, apt-file does not print a warning message.
+    </para>
+    <para>
+     Complex regular expressions that match the leading slash may not work
+     correctly. As a workaround, try to pull the leading slash to the beginning
+     of the regular expression. For example, use "/(usr/bin/vim|sbin/lvm)"
+     instead of "/usr/bin/vim|/sbin/lvm".
+    </para>
+  </refsect1>
 
+  <refsect1>
+    <title>FILES</title>
     <variablelist>
       <varlistentry>
 	<term><filename>/etc/apt/sources.list</filename></term>
@@ -441,6 +477,7 @@ Use <literal>cdrom-mount-point</literal> instead of
       </varlistentry>
     </variablelist>
   </refsect1>
+
   <refsect1>
     <title>SEE ALSO</title>
     <para>
@@ -456,29 +493,7 @@ Use <literal>cdrom-mount-point</literal> instead of
       <filename>/usr/share/doc/apt/</filename>
     </para>
   </refsect1>
-  <refsect1>
-    <title>BUGS</title>
-    <para>The cdrom backend has not been tested.</para>
-    <para>
-      Non-release lines in sources.list are not handled by apt-file.
-    </para>
-    <para>
-     There is only one Contents file per distribution that contains all
-     components (i.e. main, contrib, and non-free). Threrefore, apt-file will
-     display search results from all components, even if not all components
-     are included in the sources.list file.
-    </para>
-    <para>
-     When a new line has been added to the sources.list and apt-file update has
-     not been run, apt-file does not print a warning message.
-    </para>
-    <para>
-     Complex regular expressions that match the leading slash may not work
-     correctly. As a workaround, try to pull the leading slash to the beginning
-     of the regular expression. For example, use "/(usr/bin/vim|sbin/lvm)"
-     instead of "/usr/bin/vim|/sbin/lvm".
-    </para>
-  </refsect1>
+
   <refsect1>
     <title>AUTHOR</title>
     <para>
@@ -486,4 +501,5 @@ Use <literal>cdrom-mount-point</literal> instead of
       <email>sjg@debian.org</email>.
     </para>
   </refsect1>
+
 </refentry>
-- 
1.7.7.3

